<launch>
  <arg name="goal_idx" default="0" />
  <arg name="num_instancecs" default="1" />
  <arg name="map_file" default="$(find bwi_guidance)/maps/map3.yaml" />
  <arg name="graph_file" default="$(find bwi_guidance)/maps/graph_map3.yaml" />
  <arg name="data_dir" default="$(find bwi_guidance_solver)/data/exp1/map3/" />
  <arg name="mcts_params" 
    default="$(find bwi_guidance_solver)/config/mcts_params.json" />
  <arg name="methods_file" 
    default="$(find bwi_guidance_solver)/config/precompute_all_vi.json" />
  <node name="evaluate" pkg="bwi_guidance_solver" type="evaluate_qrr14"  
    output="screen"
    args="--map-file $(arg map_file) --graph-file $(arg graph_file)
    --data-directory $(arg data_dir) --allow-robot-current 
    --precompute-vi $(arg goal_idx) --num-instances $(arg num_instances)
    --mcts-params $(arg mcts_params) --methods-file $(arg methods_file)" 
    required="true" />
</launch>
